.gitignore

```diff
@@ -1,13 +1,20 @@
 # If a change is made in here, add it to .npmignore as well unless you
 # explicitly want to include it in the published artifact
-node_modules
+/node_modules/*
+!/node_modules/@okta
```

Comment:
Hmm, do we really have to do all of this? I think it's fine to include everything under node_modules/@okta/i18n (as long as there are no sub node_modules underneath it). We don't need to explicitly call out "everything except for the properties files" (if that's what you're doing here).